modules/core/arrow-annotations-processor/models/src/main/java/arrow/ap/objects/renzu/OptionMonoidInstance.kt

```diff
@@ -0,0 +1,11 @@
+package arrow.ap.objects.renzu
+
+import arrow.core.None
+import arrow.core.Option
+import arrow.instance
+import arrow.typeclasses.Monoid
+
+@instance(Option::class)
+interface OptionMonoidInstance<A> : OptionSemigroupInstance<A>, Monoid<Option<A>> {
```

Comment:
are these for testing purposes?

Reply:
Yep, these are just clones of real instances inside the processor models module for testing